src/config.json

```diff
@@ -803,6 +803,14 @@
     "satoshilabs.design"
   ],
   "blacklist": [
+    "mints-magiceden.live",
+    "magiceden-metaturtle.com",
+    "magiceden.watch",
+    "mints-magiceden.com",
+    "magiceden-io.digital",
+    "magiceden.autoflare.io",
```

Comment:
Hey, just wanted to let you know, as the owner of this domain this is not a phishing site.

It is a landing page I created for offering services related to Magic Eden, all the site does is basically point them to a discord server where users can create tickets for requesting services. 

At this point this site / services provided is pretty much dead / out of date, but it would be great to have it removed as it does make the domain/site seem tainted.